.github/workflows/prerelease.yaml

```diff
@@ -18,7 +18,6 @@ jobs:
             ${{ runner.os }}-node-
       - run: |
           npm install
-          npm install --force @microsoft/sarif-multitool-darwin @microsoft/sarif-multitool-win32
```

Comment:
sorry why are we removing installing multitool again?

Reply:
Rationale on #319. Hopefully that answers it.